src/system/Button/Button.tsx

```diff
@@ -65,6 +69,7 @@ const Button = forwardRef< HTMLButtonElement, ButtonProps >(
 						cursor: 'not-allowed',
 						pointerEvents: 'none',
 					},
+					width: fullWidth || fullWidth === undefined ? '100%' : 'auto',
```

Comment:
`fullWidth` had a default set to `false` previously, so it never could have been `undefined`. Assuming you only want it to be `100%` when `fullWidth` is explicitly set to `true`, this should be good.
```suggestion
					width: fullWidth ? '100%' : 'auto',
```

Reply:
It's very common in react components to pass the property as `fullWidth` without any value. We do this in other components. If I remove the false defaul value from the `fullWidth` property, and try to follow the react standard, I will have all buttons as full width:

<img width="1483" alt="image" src="https://github.com/Automattic/vip-design-system/assets/3402/8083cf36-3dbd-4f9a-ac75-27dad940bc15">

That's the reason we set it to false, so we can check for undefined, which is when someone actually passes the property without a value.

Ex:

```jsx
<Button fullWidth> // correct
<Button> // correct
<Button fullWidth={true}> // correct but not the standard
<Button fullWidth={false}> // correct but you can simply omit the prop
```

Reply:
`<Button fullWidth>` is short-hand for `<Button fullWidth={ true }>` so when you do that, the value of that prop is `true`, not `undefined`.